view/templates/netfriend.tpl

```diff
@@ -1,5 +1,5 @@
 
-<div class="intro-approve-as-friend-desc">{{$approve_as}}</div>
+<div class="intro-approve-as-friend-desc">{{$approve_as1}}<br /><br />{{$approve_as2}}<br /><br />{{$approve_as3}}</div>
```

Comment:
br-tags should be avoided, maybe use an `<ul><li>` approach here? or at least split paragraphs with `<p>` or `<div>` tag?